language/move-prover/stackless-bytecode-generator/src/stackless_bytecode_generator.rs

```diff
@@ -16,53 +21,60 @@ use vm::{
 };
 
 pub struct StacklessFunction {
-    pub local_types: Vec<SignatureToken>,
+    pub local_types: Vec<Type>,
     pub code: Vec<StacklessBytecode>,
 }
 
 pub struct StacklessBytecodeGenerator<'a> {
+    module_env: &'a ModuleEnv<'a>,
     module: &'a CompiledModule,
     function_definition_view: FunctionDefinitionView<'a, CompiledModule>,
     temp_count: usize,
```

Comment:
When I was reading the code yesterday, it looked to me as if we have the invariant temp_count == local_types.size.  I was tempted to remove temp_count.

Reply:
Not worth the effort I would say. There might be scenarios where the current thing is useful, even though apparently none in the code right now.